src/static/riot/analytics/analytics.tag

```diff
@@ -0,0 +1,520 @@
+<analytics>
+    <h1>Analytics</h1>
+
+
+    <h3>Date Range</h3>
+    <div class="ui blue buttons">
+        <button class="ui button" onclick="{ time_range_shortcut }" id="this-week">This Week</button>
+        <button class="ui button" onclick="{ time_range_shortcut }" id="this-month">This Month</button>
+        <button class="ui button" onclick="{ time_range_shortcut }" id="this-year">This Year</button>
+        <button class="ui button" onclick="{ show_date_selectors }" ref="custom_date">Custom</button>
+    </div>
+
+    <div class='date-selection' ref="date_selection_container" style="display: none;">
```

Comment:
try not to use inline styles as much as possible. Add this to the `<style>` section of the tag.

Reply:
Duly noted.